src/content-handlers/iiif/modules/uv-dialogues-module/SettingsDialogue.ts

```diff
@@ -78,6 +81,34 @@ export class SettingsDialogue extends Dialogue<
 
     this._createAccessibilityMenu();
 
+    // Extend Thumbnail Labels
+    this.$extendThumbnailLabels = $('<div class="setting extendThumbnailLabels"></div>');
+    this.$scroll.append(this.$extendThumbnailLabels);
+
+    this.$extendThumbnailLabelsCheckbox = $(
+      '<input id="extendThumbnailLabels" type="checkbox" tabindex="0" />'
+    );
+
+    this.$extendThumbnailLabels.append(this.$extendThumbnailLabelsCheckbox);
+
+    this.$extendThumbnailLabelsLabel = $(
+      '<label for="extendThumbnailLabels">' + this.content.extendThumbnailLabels  + "</label>"
+    );
+
+    this.$extendThumbnailLabels.append(this.$extendThumbnailLabelsLabel);
+
+    this.$extendThumbnailLabelsCheckbox.change(() => {
+      const settings: ISettings = {};
+
+      if (this.$extendThumbnailLabelsCheckbox.is(":checked")) {
```

Comment:
Could be simplified to: `settings.extendThumbnailLabels = this.$extendThumbnailLabelsCheckbox.is(":checked");`